WordPress/WordPressUITests/Screens/Editor/StoryPageScreen.swift

```diff
@@ -0,0 +1,58 @@
+import UITestsFoundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let backButton = "Back"
+    static let createStoryPostButton = "AnnouncementsContinueButton"
+    static let mediaPickerButton = "Media Picker Button"
+    static let closeIconButton = "Close Button"
+}
+
+class StoryPageScreen: BaseScreen {
+    let backButton: XCUIElement
+    let createStoryPostButton: XCUIElement
+    let mediaPickerButton: XCUIElement
+    let closeIconButton: XCUIElement
+    static var isVisible: Bool {
+        let app = XCUIApplication()
+        let mediaPickerButton = app.buttons[ElementStringIDs.mediaPickerButton]
+        return mediaPickerButton.exists && mediaPickerButton.isHittable
+    }
+    init() {
```

Comment:
While there are numerous style guides for Swift, that are adopted by individual companies, _most of them_ (I'm not saying _all_, because I definitely haven't seen all) agree that there should be a line break after the closing brace `}`, if that brace terminates a statement or the body of a declaration.

```suggestion
    }
    
    init() {
```